### ISSUE TITLE
Discussion about way to format code automatically (iOS & android)

### ISSUE THREAD

**Issue by MoOx** [wmcmahan/react-native-calendar-events#349]
Like for JS with prettier (already introduced in this codebase), I think it could be a good thing to format java & objective-c code so we don't have to think about it.

I found that [clang-format](https://clang.llvm.org/docs/ClangFormat.html) could do this for iOS code. @wmcmahan are you familiar with it and if so, what is the best way to introduce this tool in the codebase?

Prettier seems to have a java plugin https://github.com/jhipster/prettier-java so it should be easy fo java.

**Comment by wmcmahan** [comment]
Yes, good idea. I'm not sure what exists for objective-c code these days. There was OCLint, but I'm more recently familiar with SwiftLint / SwiftFormat. Although, anything would be a help. A CI pipeline would probably also help to automate checks.
